### PR TITLE
Read token type from contract, for Plasma Tokens

### PR DESCRIPTION
**Pull request by itzmeanjan**
This PR attempts to add support for subgraphs trying to read state of `isERC721` mapping in Registry contract, so that we can store token type in subgraph when processing Plasma token mapping request.

### CODE REVIEW

File: root/src/mappings/registry.ts [hunk 1]
```diff
@@ -11,6 +17,18 @@ export function handlePlasmaTokenMapped(event: TokenMapped): void {
...
+
+  // Attempting to check from `Registry` contract what kind of
+  // token it is.
+  //
+  // `tokenType` will be any of two possible values for Plasma Tokens
+  //
+  // 1. keccak256('ERC721') = 0x73ad2146b3d3a286642c794379d750360a2d53a3459a11b3e5d6cc900f55f44a
+  // 2. keccak256('ERC20') = 0x8ae85d849167ff996c04040c44924fd364217285e4cad818292c7ac37c0a345b
+  let registry = Registry.bind(Address.fromString(registryAddress))
+  entity.tokenType = (registry.isERC721(event.params.rootToken) ? '0x73ad2146b3d3a286642c794379d750360a2d53a3459a11b3e5d6cc900f55f44a' : '0x8ae85d849167ff996c04040c44924fd364217285e4cad818292c7ac37c0a345b') as Bytes
```
Comment (jdkanani) [inline root/src/mappings/registry.ts:29]: Shouldn't we just store string? "ERC20" or "ERC721"?

**Review by jdkanani — CHANGES_REQUESTED** [review]

File: root/src/mappings/registry.ts (hunk 1 above)
Comment (itzmeanjan) [reply root/src/mappings/registry.ts:29]: I'm doing it this way to keep this in sync with its POS counterpart.

Comment (reddyismav) [reply root/src/mappings/registry.ts:29]: POS is done in the similar way.

Comment (jdkanani) [reply root/src/mappings/registry.ts:29]: Okay should work for now.

**Review by jdkanani — APPROVED** [review]

**Pull request merged**